src/app/conf/2025/events/fost/page.tsx

```diff
@@ -0,0 +1,299 @@
+import { Metadata } from "next"
+import Image from "next-image-export-optimizer"
+
+import { Button } from "@/app/conf/_design-system/button"
+import { Accordion } from "@/app/conf/_design-system/accordion"
+
+import GraphQLFoundationWordmark from "../../assets/graphql-foundation-wordmark.svg?svgr"
+import { Hero, HeroStripes } from "../../components/hero"
+import { NavbarPlaceholder } from "../../components/navbar"
+import { CalendarIcon } from "../../../_design-system/pixelarticons/calendar-icon"
+import { PinIcon } from "../../../_design-system/pixelarticons/pin-icon"
+import { CtaCardSection } from "../../components/cta-card-section"
+
+import heroPhoto from "./hero-photo.webp"
```

Comment:
If we had a different photo for hero I could use this one to do a section like this one but with the obscuring just the left half of the photo, so the CNIT building stays visible.

<img width="1500" height="734" alt="Image" src="https://github.com/user-attachments/assets/85b1e22a-4aab-4c85-90de-b7abcd4badf6" />